Cauli/Florets/Inspector/Record List/InspectorTableViewController.swift

```diff
@@ -64,8 +64,10 @@ internal class InspectorTableViewController: UITableViewController {
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        let records = cauli.storage.records(InspectorTableViewController.recordPageSize, after: nil)
-        dataSource.append(records: records, to: tableView)
+        if dataSource.items.count == 0 {
```

Comment:
Apple recommends to use isEmpty for this. Probably won't make a difference, but doesn't hurt either?
https://developer.apple.com/documentation/swift/array/1688398-isempty

Reply:
Absolutely! I am wondering, why swiftlint didn't catch this. I will investigate.

Reply:
Alright. Locally it does catch it, but not on CI. Will investigate further 😊

Reply:
Oh, stewpid me. It did catch it on CI. The build on travis failed because of that. 🤦‍♂️

Reply:
And I fixed this.